src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -5380,11 +5382,12 @@ public object VisitTryStatement(TryStatementAst tryStatementAst)
                     Expression.Block(typeof(void), catchExprs),
                     Expression.Block(typeof(void), avs.RestoreAutomaticVar(), Expression.Assign(s_currentExceptionBeingHandled, oldrte)));
 
-                var tempVars = new List<ParameterExpression>(avs.GetTemps());
-                tempVars.Add(oldrte);
+                var tempVars = new ParameterExpression[2];
+                tempVars[0] = avs.GetTemps();
+                tempVars[1] = oldrte;
```

Comment:
We could use modern syntax to initialize.

Reply:
🤦 completely forgot about that 😂